src/mat/4C_mat_anisotropy.cpp

```diff
@@ -86,10 +86,19 @@ void Mat::Anisotropy::set_number_of_gauss_points(int numgp) { numgp_ = numgp; }
 void Mat::Anisotropy::read_anisotropy_from_element(
     const Core::IO::InputParameterContainer& container)
 {
+  // This method might be called even though there is no anisotropy possible for the element.
+  // In this case, we just return without doing anything.
+  using OptionalFiber = Core::IO::Noneable<std::vector<double>>;
+  const bool any_fibers_found =
+      container.get_if<OptionalFiber>("FIBER1") or container.get_if<OptionalFiber>("RAD") or
+      container.get_if<OptionalFiber>("AXI") or container.get_if<OptionalFiber>("CIR");
+  if (!any_fibers_found) return;
```

Comment:
@lauraengelhardt @amgebauer The fiber stuff is kind of crazy since functions that read fibers are called for elements that can never have them. I cannot fix all of that at once, so I opted for this temporary solution. At least it makes some convoluted call chains more explicit.